source/overview/faq.rst

```diff
@@ -58,7 +59,7 @@ To add yourself to the Mattermost company page on LinkedIn, do the following:
 1. Login to LinkedIn or create an account.
 2. Go to "Me" > "View profile"
 3. Under Experience, click on “plus” symbol and edit the following:
-  - Title: Developer (if you contributed code or created a plug-in, integration, or other enhancement), Contributor (if you've contributed without writing code--e.g. filed bug report, updated documentation, supported troubleshooting questions, proposed a feature, etc.) 
+  - Title: Developer (if you contributed code or created a plug-in, integration, or other enhancement), Contributor (if you've contributed without writing code--e.g. filed bug report, updated documentation, supported troubleshooting questions, proposed a feature, etc.), Translator (if you've helped translate Mattermost)
```

Comment:
Thoughts on making a list for the different titles, so that they're easier to find?

Or perhaps we can bold the words `Developer`, `Contributor` and `Translator` like

```
Title: **Developer**, if you contributed code or created a plug-in, integration, or other enhancement; **Contributor**, if you've contributed without writing code--e.g. filed a bug report, updated documentation, supported troubleshooting questions, proposed a feature, etc.; **Translator**, if you've helped translate Mattermost.
```